packages/blade/src/components/Table/TableToolbar.web.tsx

```diff
@@ -96,7 +96,11 @@ const _TableToolbar = ({
       >
         <BaseBox display="flex" alignItems="center" flex={1}>
           <BaseBox>
-            <Text size="medium" weight="semibold">
+            <Text
+              size="medium"
+              weight="medium"
+              color={isSelected ? 'surface.text.gray.normal' : 'surface.text.gray.subtle'}
```

Comment:
Earlier it was always the same color. Why are we changing the color now depending on whether its selected or not?

Reply:
Matching figma behaviour, it seems we missed adding this isSelected color change before.